### PR TITLE
Create nodejs.yml

### PR DESCRIPTION
**Pull request by jackblanc**

### CODE REVIEW

File: src/api/api.js [hunk 1]
```diff
@@ -27,7 +25,7 @@ async function createEvent(event) {
 
   let res;
   try {
-    res = await protectedResourceAxios.post(`${baseUrl}/protected/events/`, body);
+    res = await protectedResourceAxios.post(`${process.env.VUE_APP_API_DOMAIN}/api/v1/protected/events/`, body);
```
Comment (liammoyn) [inline src/api/api.js:28]: Can you keep the baseUrl const and just set it to this? I think that's easier to read than using this variable directly

File: .github/workflows/nodejs.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+name: Node CI
+
+on: [push]
```
Comment (bensenescu) [inline .github/workflows/nodejs.yml:3]: whats this do and should this PR be closed because of your other branch?

File: src/api/api.js (hunk 1 above)
Comment (jackblanc) [reply src/api/api.js:28]: We can build out a more complete structure when we have more routes, we shouldn't just have a const above.

File: .github/workflows/nodejs.yml (hunk 2 above)
Comment (jackblanc) [reply .github/workflows/nodejs.yml:3]: This runs unit tests on each PR, and no.

**Pull request merged**